nodes/scheduling/nodes-scheduler-about.adoc

```diff
@@ -6,42 +6,37 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
-
```

Comment:
@bergerhoffer Can/Should you change this heading to match the topic map: _About pod placement using the scheduler_?

Reply:
Good question. I'm not sure why they're different. There are quite a few files that link to this section, so I'll probably leave it alone for now. But let's think about whether one heading makes sense better over the other, and we can do an update later if we want.